doc/autoyast.md

```diff
@@ -186,34 +383,53 @@ About the `slp_discoverty` element, Agama does not support [SLP] at all?
 
 [SLP]: https://documentation.suse.com/sles/15-SP5/single-html/SLES-administration/#cha-slp
 
+| AutoYaST                         | Supported | Agama                     | Comment                              |
+| -------------------------------- | --------- | ------------------------- | ------------------------------------ |
+| do_registration                  | Yes       |                           | The section is ignored if `false`    |
+| email                            | Yes       | product.registrationEmail |                                      |
+| install_updates                  | Never     |                           |                                      |
+| reg_code                         | Yes       | product.registrationCode  |                                      |
+| reg_server                       | Planned   |                           |                                      |
+| reg_server_cert                  | Undecided |                           | It is not clear how addons will work |
```

Comment:
I am not sure how addones from comment is related to registration server certificate. I think it is more related to RMT

Reply:
Ouch, wrong line 😅